README.rst

```diff
@@ -75,8 +75,46 @@ that the tests pass and also the code follows the code style properly. The
 Configuring and Running
 =======================
 
-To be done... integrade is just starting and in the future you can see how to
-configure and run the functional tests in this section.
+Integrade can be configured to test any instance of cloudigrade. The
+**REQUIRED** environment variables are::
+
+    CLOUDIGRADE_BASE_URL # base url without http/https prefix
+    CLOUDIGRADE_TOKEN    # This is the token of the super user created for
+                         # cloudigrade
+
+The **OPTIONAL** environment variables are::
+
+    CLOUDIGRADE_API_VERSION # defaults to 'v1'
+    CLOUDIGRADE_USER        # defaults to 'admin'
+    CLOUDIGRADE_USER_PASS   # at the moment we only use the token to
+                            # authenticate, but this will be useful later
+    USE_HTTPS  # defaults to false so communication is done over http.
+               #  Set to True to use https.
+    SSL_VERIFY # defaults to false. If true make client verify certificate
+
+To run ``cloudigrade`` locally, especially if you want to run a branch that is
+not master, check out that branch and then follow the directions in the
+Cloudigrade readme for `running locally in OpenShift
+<https://github.com/cloudigrade/cloudigrade#running-locally-in-openshift>`_.
+Then, to deploy the code in that branch in particular, follow the directions
+for `deploying in-progress code to OpenShift
+<https://github.com/cloudigrade/cloudigrade#running-locally-in-openshift>`_.
+
+To run ``integrade`` against the test environment, it is necessary to log your
+local ``oc`` (the command line OpenShift client` into the test environment. You
+can do this by logging in through the web UI and in the menu opened by clicking
+on your user name, there is an option to ``Copy Login Command``. Paste this to
+the terminal to log the ``oc`` client into that OpenShift cluster.
+
+In either case, given that your ``oc`` binary is logged into the correct
+OpenShift cluster, you can collect the necessary `CLOUDIGRADE_BASE_URL` by
+inspecting the output of `oc status`.  To create the necessary super user and
+retreive a token for that user, use the cloudigrade make targets `make oc-user`
+and `make oc-user-authenticate`. Then export these values to
+`CLOUDIGRADE_TOKEN` and `CLOUDIGRADE_USER`.
```

Comment:
I assume CLOUDIGRADE_TOKEN is get by the `make oc-user-authenticate`, is that right? If yes I think the order these vars are shown could be exchanged and maybe use respectively to be more clear about what should be assigned to what.